cloud/amazon/ec2_group_facts.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/python
+#
+# This is a free software: you can redistribute it and/or modify
```

Comment:
This is not the standard header for Ansible modules. I see that a few EC2 related modules are using it, but we should keep to the standard one, such as the one you can find [in the ping module](https://github.com/ansible/ansible-modules-core/blob/devel/system/ping.py):

```
#!/usr/bin/python
# This file is part of Ansible
#
# Ansible is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# Ansible is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.```